scripts/expand_imports.py

```
@@ -0,0 +1,256 @@
+    return result.stdout
+
+
+def export_docs(git_dir, ref, target):
+    """Materialise <ref>:docs/ into target, and return <ref>:mkdocs.yml."""
+    target.mkdir(parents=True, exist_ok=True)
+    try:
+        run(["git", "--git-dir", str(git_dir), "cat-file", "-e", f"{ref}:docs"])
+    except RuntimeError:
+        raise RuntimeError(f"{git_dir.name} has no docs/ directory at {ref}")
+
+    subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)